src/app/dashboard/Missions/component/GroupAutoComplete.jsx

```diff
@@ -0,0 +1,63 @@
+import GroupWorkIcon from "@material-ui/icons/GroupWork";
+import Autocomplete, { createFilterOptions } from "@material-ui/lab/Autocomplete";
+import { Box, TextField } from "@material-ui/core";
+import React, { useState } from "react";
+import _ from "../../../utils/lodash";
+
+const GroupAutoComplete = ({ classes, groups, handleChange, selected }) => {
+  const filter = createFilterOptions({
+    stringify: (group) => group.groupDisplayName,
+  });
+
+  return (
+    <Box className={classes.row}>
+      <Autocomplete
+        id="group-search"
+        options={groups}
+        onChange={(event, newValue) => handleChange(newValue)}
+        renderInput={(params) => (
+          <TextField
+            {...params}
+            InputLabelProps={{
+              classes: {
+                root: classes.inputLabel,
+              },
+            }}
+            InputProps={{
+              ...params.InputProps,
+              type: "search",
+            }}
+            label="Search/Create a group"
+            placeholder="search group by name"
+            variant="outlined"
+          />
+        )}
+        getOptionLabel={(group) =>
+          group.tmpDisplayName ? group.tmpDisplayName : group.groupDisplayName
+        }
+        renderOption={(group) => (
+          <>
+            <GroupWorkIcon style={{ color: _.randomColor(group.groupDisplayName) }} />{" "}
```

Comment:
I don't see random colors in the design for this, is this necessary?

Reply:
I wish I could answer, this was code I was re-using, originally authored by @mat10tng .

Reply:
yes, we have random color for group, but this using the same seed, so its mean
hash -> specific color. But it is in the design

Reply:
Oh okay, I couldn't find random colors on the designs. Might have looked at the wrong frames